[BUG] The shellcheck pre-commit hook installation fails

## Bug details
### Describe the bug

The installation of the [pre-commit hooks][doc] fails with the error:

```
[ERROR] The hook `shellcheck` specifies `additional_dependencies` but is using language `script` which does not install an environment. Perhaps you meant to use a specific language?
```

  [doc]: https://github.com/saltstack-formulas/template-formula#pre-commit

### Steps to reproduce the bug
<!-- Include debug logs if possible and relevant, e.g. using `salt-minion -l debug`. -->
<!-- Alternatively, linking to Kitchen debug logs is useful, e.g. via. Travis CI. -->
<!-- Most useful is providing a failing InSpec test, which can be used to verify any proposed fix. -->

- Clone this repository
- Install **pre-commit** (e.g. `pip install pre-commit`)
- Install the pre-commit hooks
  ```sh
  bin/install-hooks
  ```

### Expected behaviour
<!-- A clear and concise description of what you expected to happen. -->

I expect the hooks to be installed without errors.

### Attempts to fix the bug
<!-- Please mention any attempts you have made to fix the bug and what the results were. -->

Updating the `shellcheck` hook from **v2.1.3** to **v2.1.5** (latest) fixes the issue, I'll open a PR.

--- a/.pre-commit-config.yaml
+++ b/.pre-commit-config.yaml
@@ -25,14 +25,13 @@ repos:
         args: [--debug]
         always_run: true
         pass_filenames: false
-  - repo: https://github.com/jumanjihouse/pre-commit-hooks
-    rev: 2.1.3
+  - repo: https://github.com/shellcheck-py/shellcheck-py
+    rev: v0.7.1.1
     hooks:
       - id: shellcheck
         name: Check shell scripts with shellcheck
         files: ^.*\.(sh|bash|ksh)$
         types: []
-        args: []
   - repo: https://github.com/adrienverge/yamllint
     rev: v1.23.0
     hooks:

